src/config/config_manager.cc

```diff
@@ -1253,7 +1253,7 @@ void ConfigManager::load(const fs::path& userHome)
     if (((temp == "ip") || (temp == "port")) && getOption(CFG_SERVER_PRESENTATION_URL).empty()) {
         throw std::runtime_error("Error in config file: \"append-to\" attribute "
                                  "value in <presentationURL> tag is set to \""
-            + temp + "\" but no URL is specified");
+            + temp.string() + "\" but no URL is specified");
     }
```

Comment:
You'd better change that to a `fmt::format` string

Reply:
agreed.